examples/sfbw/ex00_preimage/ex00_preimage.re

```diff
@@ -0,0 +1,8 @@
+module Universe = (val Snarky_universe.default());
```

Comment:
It looks like these examples build executables that do nothing

Reply:
I'm going to integrate them with the json top level functor once they're both in. I'm also making a few changes on your branch to the functor if that's alright with you.

Reply:
Sounds good. Will approve this now then, so you can merge later!